### ISSUE TITLE
Feature Request - Alarm only once

### ISSUE THREAD

**Issue by cpmarvin** [nttgin/BGPalerter#25]
Allow the user to set a configuration flag to only alert once. 

Example would be when you have a more specific route without an ripe route-object. 

In the current form the software alerts at predefined interval until the issue is resolved. As far as i understand the issue for a new more specific is resolved with adding the prefix in prefixes.yml but this requires manual intervention. At least in my case i only need to be aware of the first time a more specific is see in the routing table and then ignore. I undertand this might be a corner case thus a configurable config option would be great. 

Proposed changes:

- add configuration option to enable or disable re-alerts 

notificationNoResend: true|false 

- based on this flag , allow re-send of alerts in monitor.js
```
    _checkLastSent = (group) => {
        const lastTimeSent = this.sent[group.id];

        if (this.config.notificationNoResend) {
           return true;
        }

**Comment by cpmarvin** [comment]
I think you already keep track of the alarm , at least if i'm reading this correctly 

const lastTimeSent = this.sent[group.id];

and if we have that option flag , it should not affect the _clean 

            delete this.alerts[group.id];
            delete this.sent[group.id];

Hope i'm not reading the code wrong.

**Comment by massimocandela** [comment]
Yes, I already keep track of the alarms but the problem is exactly that the _clean must be prevented, otherwise there is no way the tool can remember if an alert already happened or not (if _clean is performed all alerts after will be treated as new).

I anyway created a compromise: in case alertOnlyOnce is set to true, I perform the _clean and remove only the data associated to the alert, but not the signature. So the tool can recorgnize that an alert already happened with the minimum possible memory footprint. For how small it is, in theory still a memory leak. Depending on the use case it may be a problem. For monitoring the prefixes of an AS, that is not a problem, for research/data analysis (e.g. on the entire address space) it can be a problem. So I set alertOnlyOnce to false by default and inform the user in the configuration file and documentation about this compromise.

This new option is implemented in a123f6c037eb267010244b969910018bc6eab9df and will go out with the next release.

**Comment by cpmarvin** [comment]
Thanks for taking the time to explain and for allowing this feature.

**Comment by massimocandela** [comment]
Released in [v1.21.0](https://github.com/nttgin/BGPalerter/releases/tag/v1.21.0)
